User Guide.md

```diff
@@ -0,0 +1,130 @@
+
+3. Run on a SSD. Based on painful experience: `geth` **cannot** catch up with the chain when running on a regular HDD. Read about it [here](https://medium.com/blockchain-studio/ethereum-client-geth-v1-9-0-released-whats-new-2b3de043ee16).
+
+4. `docker` >= 18.09 & `docker-compose` >= 1.24. Check with `docker --version` & `docker-compose --version`. If you do not have these installed yet, follow the official install instructions for [docker](https://docs.docker.com/install/) and [docker-compose](https://docs.docker.com/compose/install/).
+
+5. Current user can run docker (without adding `sudo`). Test with `docker run hello-world`. If this fails, [follow these instructions](https://docs.docker.com/install/linux/linux-postinstall/).
+
+
+# How to run
+
+Start the environment with
+```bash
+curl https://raw.githubusercontent.com/ExchangeUnion/xud-docker/master/xud.sh -o ~/xud.sh
+bash ~/xud.sh
+ 
+```
+This guides you through a setup on first run, pulls containers, starts syncing chains and opens
+```
+                           .___           __  .__   
+          ___  _____ __  __| _/     _____/  |_|  |  
+          \  \/  /  |  \/ __ |    _/ ___\   __\  |  
+           >    <|  |  / /_/ |    \  \___|  | |  |__
+          /__/\_ \____/\____ |     \___  >__| |____/
+                \/          \/         \/           
+--------------------------------------------------------------
+```
+
+The `status` command shows status of underlying clients, which is especially useful to track the sync status. If you are syncing full nodes on Testnet/Mainnet, this takes several hours or even days. All clients should show `Ready` before you continue.
+```bash
+btc	Ready
+lndbtc	Ready
+ltc	Syncing 2.40% (26887/1116528)
+lndltc	Waiting for sync
+geth	Syncing 42.61% (2489756/5842588)
+raiden	Waiting for sync
+xud	Waiting for sync
+```
+
+`xud ctl` takes [`xucli` commands](https://api.exchangeunion.com), like `getinfo`. Once everything is up and running, you can check existing orders of your connected peers with `orderbook` and issue an order, e.g. `sell 0.1 btc/dai 9998`. Check your `balance` before and after the swap to see it changing.
+
+# Beware
+
+Raiden currently requires direct channels with trading partners to swap reliably. We have a temporary check in place, that discards raiden-related orders (all pairs which include WETH, DAI...), if `xud` can't find a direct channel to the trading partner. You can switch this check off by setting `raidenDirectChannelChecks=false` in your `xud.conf`. Before you do that, read [this explainer of the issue](https://github.com/ExchangeUnion/xud/issues/1068).
+
+# Report
+
+Please report issues/bugs by running `report` from within `xud ctl`.
+
+# Tips 'n Tricks
+
+* Docker doesnt play nice with VPN's. We'll try to fix that in future, for the time being please disconnect VPNs running directly on your system.
```

Comment:
I haven't had any issues with my VPNs.

Reply:
I do, with e.g. OpenVPN which creates some routes to force traffic through the VPN which somehow interfere with `docker`. Formulated it a bit differently.